mw-webapp/src/logic/wayPage/reportsTable/WayColumns.module.scss

```diff
@@ -19,7 +19,9 @@ $listLeftMargin: 20px;
 }
 
 .flatButton {
+  width: 125px;
```

Comment:
is default 100px width is not enough?

Reply:
Done
![Screenshot 2023-12-16 215431](https://github.com/tritonJS826/masters-way/assets/103531486/dafac7b1-1380-480c-88ed-2e30f2028db0)